lib/driver.js

```diff
@@ -771,12 +769,63 @@ class XCUITestDriver extends BaseDriver {
     }
     log.debug(`Setting initial orientation to '${orientation}'`);
     try {
-      await this.proxyCommand('/orientation', 'POST', {orientation});
+      await this._executeCommandWithTimeout(B.promisify(this.proxyCommand), 'setScreenOrientation', '/orientation', 'POST', {orientation});
```

Comment:
There are 50+ times the method is called, so it makes sense to call the timeout version within `proxyCommand` itself.